plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/IgniteQueryRunner.java

```diff
@@ -0,0 +1,180 @@
+            "custkey bigint NOT NULL," +
+            "orderstatus varchar(1) NOT NULL," +
+            "totalprice double NOT NULL," +
+            "orderdate date NOT NULL," +
+            "orderpriority varchar(15) NOT NULL," +
+            "clerk varchar(15) NOT NULL," +
+            "shippriority integer NOT NULL," +
+            "comment varchar(79) NOT NULL) " +
+            " WITH ( " +
+            "primary_key = ARRAY['orderkey'])";
+
+    public static final String CREATE_REGION = " CREATE TABLE region (" +
+            "regionkey bigint NOT NULL," +
+            "name varchar(25) NOT NULL," +
+            "comment varchar(152) NOT NULL)" +
+            "WITH ( " +
+            "primary_key = ARRAY['regionkey'])";
+
+    private IgniteQueryRunner() {}
+
+    public static QueryRunner createIgniteQueryRunner(TestingIgniteServer server, TpchTable<?>... tables)
+            throws Exception
+    {
+        return createIgniteQueryRunner(server, ImmutableMap.of(), ImmutableMap.of(), ImmutableList.copyOf(tables));
+    }
+
+    public static DistributedQueryRunner createIgniteQueryRunner(
+            TestingIgniteServer server,
+            Map<String, String> extraProperties,
+            Map<String, String> connectorProperties,
+            Iterable<TpchTable<?>> tables)
+            throws Exception
+    {
+        DistributedQueryRunner queryRunner = null;
+        try {
+            queryRunner = DistributedQueryRunner.builder(createSession())
+                    .setExtraProperties(extraProperties)
+                    .build();
+
+            queryRunner.installPlugin(new TpchPlugin());
+            queryRunner.createCatalog("tpch", "tpch");
+
+            connectorProperties = new HashMap<>(ImmutableMap.copyOf(connectorProperties));
+            connectorProperties.putIfAbsent("connection-url", server.getJdbcUrl());
+            connectorProperties.putIfAbsent("allow-drop-table", "true");
+
+            queryRunner.installPlugin(new IgniteJdbcPlugin());
+            queryRunner.createCatalog("ignite", "ignite", connectorProperties);
+            copyFromTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, createSession(), tables);
+            return queryRunner;
+        }
+        catch (Throwable e) {
+            closeAllSuppress(e, queryRunner);
+            throw e;
+        }
+    }
+
+    public static void copyFromTpchTables(
+            QueryRunner queryRunner,
+            String sourceCatalog,
+            String sourceSchema,
+            Session session,
+            Iterable<TpchTable<?>> tables)
+    {
+        assertUpdate(queryRunner, session, CREATE_CUSTOM, OptionalLong.empty(), Optional.empty());
+        assertUpdate(queryRunner, session, CREATE_NATION, OptionalLong.empty(), Optional.empty());
+        assertUpdate(queryRunner, session, CREATE_REGION, OptionalLong.empty(), Optional.empty());
+        assertUpdate(queryRunner, session, CREATE_ORDERS, OptionalLong.empty(), Optional.empty());
```

Comment:
please create tables only if you loading them

Reply:
Here create table is a part of ``loading`` TPCH tables.

Reply:
Move to `TestIgniteConnectorTest`